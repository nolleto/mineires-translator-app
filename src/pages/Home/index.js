import './index.css';

import React, { useState } from 'react';

import Footer from '../../components/Footer';
import Header from '../../components/Header';
import InputText from '../../components/InputText';
import TranslatedText from '../../components/TranslatedText';

const App = () =>  {
  const [text, setText] = useState('Quanto está o litro de leite?\nQue horas são?');

  return (
    <div className='home'>
      <Header />

      <div className='home__content'>
        <InputText text={text} setText={setText} />
        <TranslatedText text={text} />
      </div>

      <Footer />
    </div>
  );
}

export default App;
