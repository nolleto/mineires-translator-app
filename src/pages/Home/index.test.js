import Home from './index';
import React from 'react';
import { shallow } from 'enzyme';

describe('Home page', () => {
  it('matches snapshot', () => {
    const snapshot = shallow(<Home />);

    expect(snapshot).toMatchSnapshot();
  });
});
