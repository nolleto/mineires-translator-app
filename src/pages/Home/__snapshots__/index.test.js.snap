// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<div
  className="home"
>
  <Header />
  <div
    className="home__content"
  >
    <InputText
      setText={[Function]}
      text="Quanto está o litro de leite?
Que horas são?"
    />
    <TranslatedText
      text="Quanto está o litro de leite?
Que horas são?"
    />
  </div>
  <Footer />
</div>
`;
